wap

```diff
@@ -3,24 +3,26 @@
 from archive import archive
 from args import args
 from contacts import Contacts
+from cancel import auto_cancel
 from datetime import datetime, timezone
 from describe import describe
 from approve import auto_approve
 from session import Session
-from discourse import Discourse, sync_group
```

Comment:
Discourse functionality is throwing some slight syntax errors. I have it commented out because to my knowledge it is not currently being used. I can amend the pull to have these not commented out if you'd prefer.